Lib/test/test_smtplib.py

```diff
@@ -851,6 +851,11 @@ def _auth_login(self, arg=None):
             self._authenticated(self._auth_login_user, password == sim_auth[1])
             del self._auth_login_user
 
+    def _auth_buggy(self, arg=None):
+        # This AUTH mechanism will 'trap' client in a neverending 334
+        # base64 encoded 'BuGgYbUgGy'
+        self.push('334 QnVHZ1liVWdHeQ==')
```

Comment:
Totally made-up AUTH mechanism whose purpose is to 'trap' the client in an infinite loop of `334` (AUTH Challenge)